src/ui/public/vislib/lib/layout/splits/column_chart/chart_title_split.js

```diff
@@ -12,10 +14,10 @@ define(function () {
     return function (selection) {
       selection.each(function (data) {
         var div = d3.select(this);
+        var parent = d3.select($(this).parents('.vis-wrapper')[0]);
```

Comment:
does this need to be wrapped in d3.select?  can we use only jquery, or d3 instead?